client/SymbolClient.py

```diff
@@ -199,6 +200,31 @@ def get_richlist_account_infos(self, page_number, page_size, mosaic_id):
 
         return account_infos
 
+    def is_https_valid(self):
+        try:
+            url = f'https://{self.node_host}:3001/node/info'
+            self.session.get(url, timeout=5)
+            return True
+        except:
+            return False
+
+    def is_wss_valid(self):
+        try:
+            url = f'wss://{self.node_host}:3001/ws'
+            ws = create_connection(url, timeout=5)
```

Comment:
please change the import to `import websocket` and this into `websocket.create_connection`
(name is pretty generic, so it's nicer to use module-qualified name here)